localisation/ja-jp/docs/01-monkey-app.md

```diff
@@ -0,0 +1,164 @@
+    mkdir -p $REPOSITORY_ROOT/workshop
+    cd $REPOSITORY_ROOT/workshop
+    dotnet new console -n MyMonkeyApp
+    ```
+
+    ```powershell
+    # PowerShell
+    New-Item -Type Directory -Path $REPOSITORY_ROOT/workshop -Force
+    cd $REPOSITORY_ROOT/workshop
+    dotnet new console -n MyMonkeyApp
+    ```
+
+## GitHubリポジトリの管理
+
+1. 作成したコンソールアプリをプッシュするために、GitHub Copilotに以下のプロンプトを入力します。
+
+    ```text
+    Push the current changes to the `main` branch of the repository.
+    ```
+
+1. リポジトリにissueを生成するために、GitHub Copilotに以下のプロンプトを入力します。
+
+    ```text
+    Create a new GitHub issue in my repository titled 'Implement Monkey Console Application' with the following requirements:
+    
+    - Create a C# console app that can list all available monkeys, get details for a specific monkey by name, and pick a random monkey.
+    - The app should use a Monkey model class and include ASCII art for visual appeal.
+    - Add appropriate labels like 'enhancement' and 'good first issue'.
+    - Add some details about how we may go about implementing this and a checklist for what we will need to do.
+    ```
+
+1. issueに`@Copilot`をアサインし、何が起こっているかを観察します。
+
+## MCPサーバーの開始 &ndash; Monkey MCP
+
+1. `F1`または Windows では`Ctrl`+`Shift`+`P`、Mac OSでは`Cmd`+`Shift`+`P`を押してコマンドパレットを開き、`MCP: List Servers`を検索します。
+1. `github` MCPサーバーが稼働していることを確認します。
+1. `monkeymcp`を選択して`Start Server`をクリックします。
+
+## GitHub CopilotとMCPサーバーでMonkeyアプリを開発
+
+1. サルのリストを取得するために以下のプロンプトを入力します。
+
+    ```text
+    Get me a list of monkeys that are available and display them in a table with their details.
+    ```
+
+1. サルのデータモデルのアイデアを得るために以下のプロンプトを入力します。
+
+    ```text
+    What would a data model look like for this structure?
+    ```
+
+1. データモデルクラス用のファイルを作成するために以下のプロンプトを入力します。
+
+    ```text
+    Let's create a new file for this class.
+    ```
+
+1. `MonkeyHelper`クラスを作成するために以下のプロンプトを入力します。
+
+    ```text
+    Let's create a new class called MonkeyHelper that is static. It should manage a collection of monkey data. Include methods to get all monkeys, get a random monkey, find a monkey by name, and track access count to when a random monkey is picked. The data for the monkeys should come from the Monkey MCP server that we just got.
+    ```
+
+1. コンソールアプリを更新するために以下のプロンプトを入力します。
+
+    ```text
+    Let's update the app now to have a nice menu with the following options that will call into that `MonkeyHelper`.
+    
+    1. List all monkeys
+    2. Get details for a specific monkey by name
+    3. Get a random monkey
+    4. Exit app
+
+    Also display some funny ASCII art randomly.
+    ```
+
+1. 更新されたコンソールアプリをプッシュするために、GitHub Copilotに以下のプロンプトを入力します。
+
+    ```text
+    Push the current changes to the `mymonkeyapp` branch of the repository.
+    With this branch, create a PR against the `main` branch.
+    Connect this PR to the issue created before.
+    Then, merge this PR and close the issue.
+    ```
+
+---
+
+OK。「MCPを使ったMonkeyアプリ開発」ステップが完了しました。[STEP 02: MCPサーバー](./02-mcp-server.md)に進みましょう。
```

Comment:
Make sure that the disclaimer should be included at the end of the localised document.